chrome/openshift-navigation.json

```diff
@@ -33,6 +33,24 @@
             "groupId": "insights",
             "title": "Insights",
             "navItems": [
+                {
```

Comment:
This item was not added before for some reason (even when the app has been normally functioning before in prod/beta). I have copy-pasted this new object item from ci-beta version https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/openshift-navigation.json#L41.